src/components/calcite-switch/calcite-switch.tsx

```diff
@@ -97,7 +104,7 @@ export class CalciteSwitch {
   /** Sets focus on the component. */
   @Method()
   async setFocus(): Promise<void> {
-    focusElement(this.inputEl);
+    this.el.focus();
```

Comment:
For this one, i changed it to focus on the host because the input is hidden so it has no visual state for being focused and can't be tabbed to. We should probably just get rid of this hidden input and use aria roles here.

Reply:
> We should probably just get rid of this hidden input and use aria roles here.

Agreed. I updated `Host` accordingly. I'll submit an issue to clean this (and others?) up since we're [not supposed to override user-set global attributes](https://developers.google.com/web/fundamentals/web-components/best-practices#do-not-override-author-set,-global-attributes.). Regarding the hidden input, let's remove it when we implement `FormComponent` across the board.